flask_unchained/bundles/cors/extensions/cors.py

```diff
@@ -0,0 +1,6 @@
+from flask_cors import CORS as BaseCORS
+
+
+class CORS(object):
```

Comment:
I don't think this class is necessay - upstream's extension code seems to work as-is.